content/keps.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Collaboration with Kubernetes SIG-Node"
+---
+
+containerd is a popular container runtime used with
+[Kubernetes](https://kubernetes.io).  containerd implements the Kubernetes
+Container Runtime Interface (CRI) and is expected to participate in the
+development and implementation of CRI.
+
+Changes to Kubernetes (including CRI) are primarily driven through the
+Kubernetes Enhancement Proposal (KEP) process.  KEPs are proposed to individual
+Special Interest Groups (SIGs) within the Kubernetes project, which help to
+refine, clarify, and decide the content of an individual KEP as well as drive
+its implementation.  CRI-related KEPs are primarily handled through SIG-Node.
+SIG-Node has
+[adopted a policy](https://www.kubernetes.dev/docs/code/cri-api-dev-policies/)
+which requires KEPs to reach maturity in both containerd and CRI-O before
+advancing through graduation stages.
+
+This document describes the process that the containerd project uses to
+collaborate with SIG-Node on any KEP or other CRI-related change.
+
+## Goals
+
+* Provide visibility of SIG-Node integration work for containerd (including
+  KEPs) for:
+  * containerd maintainers and community members who are not directly involved
+    in SIG-Node
+  * SIG-Node participants who are not involved in the containerd community
+* Ensure containerd maintainers and communtiy members have adequate
+  opportunities to participate in design without requiring full context on
+  every SIG-Node KEP or who are not familiar with the KEP process
+* Ensure contributors understand the status of their proposed contributions
+* Spread the KEP tracking and review load among interested maintainers, and
+  enable at least two interested containerd maintainers per KEP or
+  Kubernetes-related work item in order to facilitate knowledge sharing
+
+## Roles
+
+* SIG-Node member liaison - a containerd maintainer responsible for
+  facilitating communication between SIG-Node and the containerd project
+  regarding KEP status, targeted containerd releases, and any other integration
+  work or pain points.
+* KEP shepherd - for each KEP, at least one (preferably two) containerd
+  maintainer(s) responsible for helping a KEP move through the implementation
+  and release process in containerd.
+
+## Metadata
+
+* Per-KEP issue in the
+  [containerd/containerd](https://github.com/containerd/containerd) repository
+  for tracking KEPs that require CRI changes in the container
+  runtime or one of our other containerd subprojects.  (This includes
+  relatively minor changes, such as a runc update or checking containerd config
+  to align defaults with feature gate GAs.)
+* Labels within the containerd/containerd repository corresponding to
+  Kubernetes minor releases, applied to KEP tracking issues and PRs
+* Recording the assigned shepherd(s) on the KEP tracking issue
+
+## Process
+
+For a KEP to be reviewed and implemented in containerd:
+
+### Issue management
+
+* An issue should be created in the
```

Comment:
sorry for the late feedback. One recommendation I have is to avoid having issues titled "KEP NNNN". Instead have it be named "KEP NNNN: this is what this do" (make issues title be descriptive enough)

Reply:
done